routes/users.js

```diff
@@ -1,9 +1,9 @@
-var express = require('express');
+var express = require("express");
 var router = express.Router();
```

Comment:
```suggestion
const express = require("express");
const router = express.Router();
```
해당 부분도 `const`로 동일하게 바꿔주시면 좋을 것 같습니다.